### DISCUSSION TITLE
Update to Studio drivers vs Game Ready drivers

### DISCUSSION THREAD

**Discussion by D-AA-VID** [ElPumpo/TinyNvidiaUpdateChecker#145] [Ideas]
Hi,
Happy new year and thank you for this utility !!!
Could it be possible to update to Studio drivers beside Game Ready drivers with a command line argument ?
Best regards
David

**Comment by HawaiiBeach** [comment]
I belive the studio drivers are not available in the AJAX NVIDIA API. I would have to do it like in the old days which is not the best. But I could implement it if there's enough people who want it.

**Reply by marcelock** [reply]
Checking if it is already be possible to update to Studio drivers now.

**Reply by HawaiiBeach** [reply]
I have not done any research on that API I am using. So I don't know.

**Comment by HawaiiBeach** [comment]
Fresh research is just in. The AJAX API does have support for studio drivers! Follow #187

**Comment by HawaiiBeach** [comment]
Added in v1.17.0!
